packages/react-components/react-calendar-compat/src/components/Calendar/Calendar.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import type { ForwardRefComponent } from '@fluentui/react-utilities';
+import { useCalendar_unstable } from './useCalendar';
+import { renderCalendar_unstable } from './renderCalendar';
+import { useCalendarStyles_unstable } from './useCalendarStyles.styles';
+import type { CalendarProps } from './Calendar.types';
+
+/**
+ * Calendar component - TODO: add more docs
+ */
+export const Calendar: ForwardRefComponent<CalendarProps> = React.forwardRef((props, ref) => {
+  const state = useCalendar_unstable(props, ref);
+
+  useCalendarStyles_unstable(state);
```

Comment:
I would recommend you create the component in a separate PR since you should add custom style hooks, and that would require an update to the types of react-shared-contexts.

This will be automated after https://github.com/microsoft/fluentui/pull/29452